manifests/deis-router-rc.yaml

```diff
@@ -32,3 +32,15 @@ spec:
           hostPort: 2222
         - containerPort: 9090
           hostPort: 9090
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 9090
+          initialDelaySeconds: 1
+          timeoutSeconds: 1
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 9090
+          initialDelaySeconds: 1
```

Comment:
I _think_.  In my experience, router is ready in a lot faster than 30 seconds.  I honestly took the 30 seconds from workflow's probes and I _know_ this starts faster than workflow.  I will be happy to increase these values over time if they turn out to be too small.